db/migrate/20230509171550_change_likes_user_id_to_author_id.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeLikesUserIdToAuthorId < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :likes, :user_id, :author_id
+  end
+end
```

Comment:
- [ ] @chepkok3 Great work implementing the changes requested, however, you added an extra migration for renaming the user_id column in the likes table which doesn't exist, this change was done in the migration file `20230509144722_add_user_ref_to_likes.rb` , kindly delete it so that your migrations can run successfully,

Reply:
@danmainah Anything else or just deleting the 20230509144722 migration file?